src/cloud/services/UserService.ts

```diff
@@ -23,8 +28,197 @@ const findUserById = async (id: string): Promise<Parse.User> => {
   }
 };
 
+const findUserAdmin = async (displayLimit: number, page: number): Promise<Parse.Object[]> => {
```

Comment:
Review function names, and check if the user is admin based on the current user.